### PR TITLE
Dataset/lv business register

### PR DESCRIPTION
**Pull request by simonwoerpel**

### CODE REVIEW

File: datasets/lv_business_register/parse.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+def parse_register(context: Zavod, row: dict):
+    company = context.make("Company")
+    company.id = company_id(row["regcode"])
+    company.add("name", row["name"])
+    company.add("legalForm", row["type_text"])
+    company.add("incorporationDate", row["registered"])
+    company.add("address", row["address"])
+    company.add("opencorporatesUrl", oc_url(row["regcode"]))
+    company.add("jurisdiction", "lv")
+    company.add("country", "lv")
```
Comment (pudo) [inline datasets/lv_business_register/parse.py]: Don't need this, I think.

File: datasets/lv_business_register/parse.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+    if row["terminated"]:
+        company.add("dissolutionDate", row["terminated"])
+        company.add("status", row["closed"])
```
Comment (pudo) [inline datasets/lv_business_register/parse.py]: Let's unindent

File: datasets/lv_business_register/parse.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+
+def parse_beneficial_owners(context: Zavod, row: dict):
+    officer = make_officer(context, row)
+    officer.add("nationality", row["nationality"])
+    officer.add("country", row["residence"])
+    cid = company_id(row["legal_entity_registration_number"])
+    rel = context.make("Ownership")
+    rel.id = context.make_slug("OWNER", officer.id, cid)
+    rel.add("role", "OWNER")
```
Comment (pudo) [inline datasets/lv_business_register/parse.py:139]: is this uppercase OWNER from the source? I wouldn't want to make ti up

**Review by pudo — CHANGES_REQUESTED** [review]

File: datasets/lv_business_register/parse.py (hunk 3 above)
Comment (simonwoerpel) [reply datasets/lv_business_register/parse.py:139]: yes, there are 3 source files, the "officers.csv" which is basically Directorships but in some cases, when the type is OWNER, its an Ownership... this uppercase type comes from the source
then, in the 2 other files, "members.csv" and "beneficial_owners.csv" which specify the Ownership connection a bit more, i use the uppercase OWNER to generate the same id for the connection (and add role prop to be consistent with the previously created relations)

File: datasets/lv_business_register/parse.py (hunk 2 above)
Comment (simonwoerpel) [reply datasets/lv_business_register/parse.py]: :heavy_check_mark:

File: datasets/lv_business_register/parse.py (hunk 1 above)
Comment (simonwoerpel) [reply datasets/lv_business_register/parse.py]: u mean country prop?

**Pull request merged**